Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Task.java

```diff
@@ -657,4 +863,61 @@ public boolean equals(Object object) {
     public int hashCode() {
         return Objects.hash(title, processingTime, processingBegin, processingEnd, process, template);
     }
+
+    /**
+     * Returns the translated processing status of the task at runtime. This can
+     * be translated at runtime for the user currently displaying this task. Can
+     * be {@code null} if currently no value is available.
+     *
+     * @return translated processing status
+     */
+    public String getProcessingStatusTitle() {
+        return processingStatusTitle;
+    }
+
+    /**
+     * Sets the translated processing status of the task at runtime. This is a
+     * transient value that is not persisted.
+     *
+     * @param processingStatusTitle
+     *            translated processing status to set
+     */
+    public void setProcessingStatusTitle(String processingStatusTitle) {
+        this.processingStatusTitle = processingStatusTitle;
+    }
+
+    /**
+     * Returns the translated processing type of the task at runtime. This can
+     * be translated at runtime for the user currently displaying this task. Can
+     * be {@code null} if currently no value is available.
+     *
+     * @return translated processing status
+     */
+    public String getEditTypeTitle() {
+        return editTypeTitle;
+    }
+
+    /**
+     * Sets the translated processing type of the task at runtime. This is a
+     * transient value that is not persisted.
+     *
+     * @param editTypeTitle
+     *            translated processing type to set
+     */
+    public void setEditTypeTitle(String editTypeTitle) {
+        this.editTypeTitle = editTypeTitle;
+    }
+
+    /**
+     * Returns a list of the IDs of the roles, whose holders are allowed to
+     * perform this task. This list is not guaranteed to be in reliable order.
+     *
+     * @return list of the IDs of the roles
+     */
+    public List<Integer> getRoleIds() {
+        if (Objects.isNull(roles)) {
+            return null;
```

Comment:
Would it not be better instead of returning `null` here returning an empty list?